styleguide/labels-projects.md

```diff
@@ -0,0 +1,118 @@
+# Labels and projects roadmap
+
+The .NET docs team makes extensive use of [GitHub labels](https://github.com/dotnet/docs/labels) to organize our work. By filtering on combinations of labels, we can quickly focus on sections of the [.NET docs website](https://docs.microsoft.com/dotnet) of interest. 
+
+We also use [GitHub projects](https://github.com/dotnet/docs/projects) to organize sprints and other goal-oriented epics.
+
+This roadmap explains how we use these organizational tools, and has links to handy filters we use to find areas of interest.
+
+## Labels
+
+If this is your first experience with [dotnet/docs](https://github.com/dotnet/docs), start with the [up-for-grabs](https://github.com/dotnet/docs/labels/up-for-grabs) issues. These are issues that have a more focused scope. They are a great way to make your first contribution. From the up-for-grabs view, you can further filter issues based on areas and priority.
+
+We use labels to classify issues in many different ways:
+
+- [.NET Guides](find-a-single-net-guide) and [sections of a guide](#search-one-section-of-a-guide).
+- [Target release](#releases)
+- [Priority](#priority)
+
+You can combine a label from each set (guide, release, priority) to create a narrow focus to find issues you want to work on.
+
+### Find a single .NET guide
+
+We use labels for each of the architecture guides, and for each .NET Guide. 
+
+![:book: guide on light green background](./images/guide.png "Prefix for architecture guide labels")
+
+Architecture guides are noted with the `:book: guide` prefix and have a light green background. These are the long-form areas that cover recommended architecture. Here are current issues filtered for each of the .NET architecture guides.
+
+- [ASP.NET Core web apps](https://github.com/dotnet/docs/labels/%3Abook%3A%20guide%20-%20ASP.NET%20Core%20web%20apps)
+- [Blazor](https://github.com/dotnet/docs/labels/%3Abook%3A%20guide%20-%20Blazor)
+- [Cloud Native](https://github.com/dotnet/docs/labels/%3Abook%3A%20guide%20-%20Cloud%20Native)
+- [Docker lifecycle](https://github.com/dotnet/docs/labels/%3Abook%3A%20guide%20-%20Docker%20lifecycle)
+- [Framework design guidelines](https://github.com/dotnet/docs/labels/%3Abook%3A%20guide%20-%20Framework%20Design%20Guidelines)
```

Comment:
Isn't it the copyrighted content where contributions are not allowed/welcome? Entry in this list looks like an invitation to contribute.

Reply:
It is. I'll add a note regarding that.